cli/config/configfile/file.go

```diff
@@ -330,6 +332,36 @@ func (configFile *ConfigFile) GetFilename() string {
 	return configFile.Filename
 }
 
+// PluginConfig retrieves the requested option for the given plugin.
+func (configFile *ConfigFile) PluginConfig(pluginname, option string) (string, bool) {
```

Comment:
Can we update the helloworld example to use these primitives? Appart from reading the unit tests, there is no obvious  way to be aware of this feature.

Reply:
Sure, I'll taker a look ASAP

Reply:
I did that, and made a unit test to cover these paths.